Tips-for-common-editors.md

```diff
@@ -0,0 +1,141 @@
+**A word of warning: Many code editors let you perform git actions from their graphical interfaces instead of through the command line. We strongly recomment that you use the command line instead. Many new contributors run into trouble because their editor's git interface is incorrectly configured.**
```

Comment:
```suggestion
**A word of warning: Many code editors let you perform git actions from their graphical interfaces instead of through the command line. We strongly recommend that you use the command line instead. Many new contributors run into trouble because their editor's git interface is incorrectly configured.**
```

Reply:
done